src/ontology/external/mondo-omim-genes.robot.tsv

```diff
@@ -294,7 +293,6 @@ http://purl.obolibrary.org/obo/MONDO_0008019	http://identifiers.org/hgnc/12783	O
 http://purl.obolibrary.org/obo/MONDO_0008024	http://identifiers.org/hgnc/14025	OMIM:158580	https://omim.org/entry/608761
 http://purl.obolibrary.org/obo/MONDO_0008025	http://identifiers.org/hgnc/30171	OMIM:158590	https://omim.org/entry/608014
 http://purl.obolibrary.org/obo/MONDO_0008026	http://identifiers.org/hgnc/2961	OMIM:158600	https://omim.org/entry/600112
-http://purl.obolibrary.org/obo/MONDO_0008031	http://identifiers.org/hgnc/29090	OMIM:158901	https://omim.org/entry/614982
```

Comment:
NOTE: This should not have been removed. See https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L4

Reply:
all missing information is fixed. see [here](https://github.com/monarch-initiative/mondo-ingest/pull/808/files#r1990405408) for details